UHPostalService/UHPostalService/Pages/Account/Customers/Edit.cshtml.cs

```diff
@@ -38,12 +38,13 @@ public async Task<IActionResult> OnGetAsync(int? id)
             {
                 return NotFound();
             }
-           ViewData["AddressID"] = new SelectList(_context.Addresses, "Id", "City");
+           ViewData["Address.City"] = new SelectList(_context.Addresses, "Id", "City");
```

Comment:
u don't actually use a selectlist anymore since you're updating the whole address, not giving the option to select from a list